DuckDuckGo/OnboardingExperiment/OnboardingBackground.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  OnboardingBackground.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
+struct OnboardingBackground: View {
+
+    var body: some View {
+        ZStack {
+            Gradient()
+
+            Image(.daxOnboardingBackground)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+        }
+        .ignoresSafeArea()
+    }
+}
+
+// MARK: - Gradient
+
+private extension OnboardingBackground {
+
+    struct Gradient: View {
+        @Environment(\.colorScheme) private var colorScheme
+
+        var body: some View {
+            switch colorScheme {
+            case .light:
+                lightGradient
+            case .dark:
+                darkGradient
+            @unknown default:
+                lightGradient
+            }
+        }
+
+        private var lightGradient: some View {
```

Comment:
Should we put these colours in the asset catalogue?

Reply:
Nah, these seem really specific to the onboarding and we'd just be generating a bunch of assets to avoid having to use the colorScheme - this seems like a reasonable exception to the rule imo.